api/v1alpha1/manifest_types.go

```diff
@@ -54,53 +48,23 @@ type ManifestSpec struct {
 
 	// Config specifies OCI image configuration for Manifest
 	// +kubebuilder:validation:Optional
-	Config types.ImageSpec `json:"config"`
+	Config types.ImageSpec `json:"config,omitempty"`
```

Comment:
isnt't +kubebuilder:validation:Optional = omitempty? Maybe we can standardize it across the api defn.

Reply:
afaik omitempty only removes the field from the Storage version if not found. Maybe it has the same effect, will double check